scala/conversion/src/org/jetbrains/plugins/scala/conversion/visitors/SimplePrintVisitor.scala

```diff
@@ -790,10 +790,17 @@ class SimplePrintVisitor protected() {
     }
   }
 
-  protected def visitSwitchLabelStatement(caseValue: Option[IntermediateNode], arrow: String): Unit = {
+  protected def visitSwitchLabelStatement(caseValues: Seq[IntermediateNode], arrow: String,
+                                          body: Option[IntermediateNode]): Unit = {
     printer.append("case ")
-    if (caseValue.isDefined) visit(caseValue.get)
+    var first = true
+    caseValues.foreach(e => {
```

Comment:
There is a `printWithSeparator` utility method.

Reply:
I'm going to cherry-pick it manually, so I'll fix it.

Reply:
Thank you